submissions/Oleksandr-Heleta/popup/style.css

```diff
@@ -0,0 +1,226 @@
+* {
+    margin: 0;
+    padding: 0;
+
+}
+
+a {
+    text-decoration: none;
+    color: #757575;
+
+}
+
+li {
+    list-style: none;
+    cursor: pointer;
+
+}
+
+
+.popup_more-input,
+.popup_button-input {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+
```

Comment:
No need to add an empty line before closing brackets)

Reply:
Have already deleted